borg/archiver.py

```diff
@@ -1353,6 +1398,11 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('--keep-tag-files', dest='keep_tag_files',
                                action='store_true', default=False,
                                help='keep tag files of excluded caches/directories')
+        subparser.add_argument('--pattern', action=ArgparsePatternAction,
+                               metavar="PATTERN", help='include/exclude paths matching PATTERN')
```

Comment:
this option might be repeated quite some times, so it would be nice to have a (non-conflicting) short option here.